insights/parsers/nvme_core_io_timeout.py

```diff
@@ -0,0 +1,39 @@
+"""
+NVMeCoreIOTimeout - The timeout for I/O operations submitted to NVMe devices
+============================================================================
+
+This parser reads the content of ``/sys/module/nvme_core/parameters/io_timeout``.
+"""
+
+from insights import Parser, parser
+from insights.specs import Specs
+from ..parsers import SkipException, ParseException
+
+
+@parser(Specs.nvme_core_io_timeout)
+class NVMeCoreIOTimeout(Parser):
+    """
+    Class for parsing the content of the ``/sys/module/nvme_core/parameters/io_timeout``.
+
+    A typical sample of the content of this file looks like::
+
+        30
```

Comment:
Better use the same value with the one in `Examples`. (line-31)

Reply:
Okay, got it.